controller/Lists.php

```diff
@@ -319,10 +323,13 @@ public function saveLists(ValueCollectionService $valueCollectionService): void
             )
         );
 
-        $listElements = array_filter($payload, function (string $key) {
-            return (bool)preg_match('/^list-element_/', $key);
-        },
-        ARRAY_FILTER_USE_KEY);
+        $listElements = array_filter(
+            $payload,
+            function (string $key) {
```

Comment:
`function (string $key): bool`

Reply:
No problem, but you know, I did not touch it most of the changes related to cs fixer formatting.